### PR TITLE
main/atk: upgrade to 2.32.0

### PR DESCRIPTION
**Pull request by Cogitri**
* modernize APKBUILD
* adopt maintainership

### CODE REVIEW

File: main/at-spi2-atk/APKBUILD [hunk 1]
```diff
@@ -1,45 +1,37 @@
...
-pkgrel=1
+pkgver=2.32.0
+pkgrel=0
 pkgdesc="A GTK+ module that bridges ATK to D-Bus at-spi"
 url="https://www.freedesktop.org/wiki/Accessibility/AT-SPI2/"
 arch="all"
 options="!check"  # Requires dbus daemon running.
 license="LGPL-2.0-or-later"
-depends=""
-depends_dev="dbus-dev atk-dev"
```
Comment (tcely) [inline main/at-spi2-atk/APKBUILD]: Why should `-dev` no longer have these dependencies?

Comment (Cogitri) [reply main/at-spi2-atk/APKBUILD]: abuild warned me that these are already automagically added. Will add a notice to the commit msg.

File: main/at-spi2-core/APKBUILD [hunk 2]
```diff
@@ -1,40 +1,34 @@
...
+pkgver=2.32.0
+pkgrel=0
 pkgdesc="Protocol definitions and daemon for D-Bus at-spi"
-url="http://www.linuxfoundation.org/en/AT-SPI_on_D-Bus"
+url="https://www.freedesktop.org/wiki/Accessibility/AT-SPI2/"
 arch="all"
 options="!check"  # Requires running dbus daemon.
 license="LGPL-2.0-or-later"
-depends=""
-depends_dev="libxtst-dev"
```
Comment (tcely) [inline main/at-spi2-core/APKBUILD]: I would think `-dev` should depend on `dbus-dev` at least.

File: main/at-spi2-core/APKBUILD [hunk 3]
```diff
@@ -1,40 +1,34 @@
...
-makedepends="$depends_dev dbus dbus-dev glib-dev intltool gobject-introspection-dev
-	meson"
-install=""
+makedepends="libxtst-dev dbus dbus-dev glib-dev gobject-introspection-dev meson"
+checkdepends="dbus"
```
Comment (tcely) [inline main/at-spi2-core/APKBUILD]: `dbus` is already in `makedepends` so this is not needed.

File: main/atk/APKBUILD [hunk 4]
```diff
@@ -1,38 +1,36 @@
...
-depends=""
 makedepends="glib-dev gobject-introspection-dev musl-utils meson ninja gtk-doc"
 subpackages="$pkgname-dev $pkgname-doc $pkgname-lang"
 source="https://download.gnome.org/sources/$pkgname/${pkgver%.*}/$pkgname-$pkgver.tar.xz"
-builddir="$srcdir/$pkgname-$pkgver"
 
 build() {
-	mkdir -p "$builddir"/build
-	cd "$builddir"/build
+	mkdir -p "$builddir"
```
Comment (tcely) [inline main/atk/APKBUILD]: Is this actually needed?

File: main/at-spi2-atk/APKBUILD (hunk 1 above)
Comment (tcely) [reply main/at-spi2-atk/APKBUILD]: Thank you

File: main/at-spi2-core/APKBUILD (hunk 2 above)
Comment (Cogitri) [reply main/at-spi2-core/APKBUILD]: Huh, looking at abuild output it detects `pc:dbus-1>=1.5` just fine as dep

File: main/at-spi2-core/APKBUILD (hunk 3 above)
Comment (Cogitri) [reply main/at-spi2-core/APKBUILD]: Thanks

File: main/atk/APKBUILD (hunk 4 above)
Comment (Cogitri) [reply main/atk/APKBUILD]: Ah, no.

**Pull request closed without merging**